src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+const Button = props => {
```

Comment:
**[OPTIONAL]**
`const Button = ({name}) => {} ` will save you from writing line 4